### ISSUE TITLE
Endless loading in the UI if the birthdate of a person is updated, or a case is transferred from home into a facility

### ISSUE THREAD

**Issue by adinaflorea9** [hzi-braunschweig/SORMAS-Project#10425]
### Bug Description

Making changes on a person page and clicking 'Save' leads to an endless loading. No error is displayed in the server logs or in the UI.

In the 'Cases' directory, if a case's 'Place of stay' is updated from 'home' to facility, an endless loading appears as well.

### Steps to Reproduce

I:
1. Open a person and set the 'Date of birth';
2. Click 'Save'.

II:
1. Open a case that has the 'Place of stay' set to home;
2. Update the 'Place of stay' to 'Facility';
3. Click 'Save' in the 'Current hospitalization' pop-up that is triggered after step 2.

### Actual Behavior

The UI displays an endless loading screen. No error is displayed in the UI or in the server logs. After stopping and restarting the server, it is visible that the changes made on the person page/or the case page were not saved.

### Expected Behavior

The changes made on the person page. or the case page should be saved and no endless loading should happen.

### Screenshots

GIF of the steps and the behavior:
![Persons endless server loop-loading](https://user-images.githubusercontent.com/96909231/191722306-f6cf86d7-d14b-42fd-9407-116ef076e0c2.gif)

### System Details

* Device: Windows 10
* SORMAS version: 1.76.0-SNAPSHOT(90f6723)
* Android version/Browser: Chrome
* Server URL: local machine
* User Role: Admin+National User

### Additional Information

The behavior could not be reproduced on older versions of SORMAS, but it was reproducible for @roxanamlendea on her local machine as well and on the testing instances after the build and deploy to test that took place at noon on the 22nd of September.

I logged a separate ticket as I could not determine which ticket from the current sprint might have caused the issue.

**Comment by MateStrysewske** [comment]
Found the source of the problem: When changing a person, the `onCaseChanged` method is called twice. Both times, the `clearCompleteness` method is then called, which has a `@TransactionAttribute(TransactionAttributeType.REQUIRES_NEW)` annotation which likely causes a database deadlock. Removing the annotation gets rid of the problem.

Wondering whether this annotation is really needed there - @StefanKock @MartinWahnschaffe let's discuss whether we can just get rid of it or whether we need to find a different solution.

**Comment by StefanKock** [comment]
Problem was introduced with  #8502 in 1.76.0, so this ticket is definitely to be discarded and causing ticket to be reopened.

**Comment by adinaflorea9** [comment]
Ticket #8502 was reopened. Ticket #10425 has been discarded.
